content/upload/upload_1.md

```diff
@@ -0,0 +1,413 @@
+	BR_FEATURES_DIRECTOR BOOLEAN,
+	BR_FEATURES_QUICKTIME BOOLEAN,
+	BR_FEATURES_REALPLAYER BOOLEAN,
+	BR_FEATURES_WINDOWSMEDIA BOOLEAN,
+	BR_FEATURES_GEARS BOOLEAN,
+	BR_FEATURES_SILVERLIGHT BOOLEAN,
+	BR_COOKIES BOOLEAN,
+	BR_COLORDEPTH VARCHAR(12),
+	BR_VIEWWIDTH NUMBER(38,0),
+	BR_VIEWHEIGHT NUMBER(38,0),
+	OS_NAME VARCHAR(50),
+	OS_FAMILY VARCHAR(50),
+	OS_MANUFACTURER VARCHAR(50),
+	OS_TIMEZONE VARCHAR(255),
+	DVCE_TYPE VARCHAR(50),
+	DVCE_ISMOBILE BOOLEAN,
+	DVCE_SCREENWIDTH NUMBER(38,0),
+	DVCE_SCREENHEIGHT NUMBER(38,0),
+	DOC_CHARSET VARCHAR(128),
+	DOC_WIDTH NUMBER(38,0),
+	DOC_HEIGHT NUMBER(38,0),
+	TR_CURRENCY VARCHAR(3),
+	TR_TOTAL_BASE NUMBER(18,2),
+	TR_TAX_BASE NUMBER(18,2),
+	TR_SHIPPING_BASE NUMBER(18,2),
+	TI_CURRENCY VARCHAR(3),
+	TI_PRICE_BASE NUMBER(18,2),
+	BASE_CURRENCY VARCHAR(3),
+	GEO_TIMEZONE VARCHAR(64),
+	MKT_CLICKID VARCHAR(128),
+	MKT_NETWORK VARCHAR(64),
+	ETL_TAGS VARCHAR(500),
+	DVCE_SENT_TSTAMP TIMESTAMP_NTZ(9),
+	REFR_DOMAIN_USERID VARCHAR(128),
+	REFR_DVCE_TSTAMP TIMESTAMP_NTZ(9),
+	DOMAIN_SESSIONID VARCHAR(128),
+	DERIVED_TSTAMP TIMESTAMP_NTZ(9),
+	EVENT_VENDOR VARCHAR(1000),
+	EVENT_NAME VARCHAR(1000),
+	EVENT_FORMAT VARCHAR(128),
+	EVENT_VERSION VARCHAR(128),
+	EVENT_FINGERPRINT VARCHAR(128),
+	TRUE_TSTAMP TIMESTAMP_NTZ(9),
+	LOAD_TSTAMP TIMESTAMP_NTZ(9),
+	CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_UA_PARSER_CONTEXT_1 VARCHAR,
+	CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_WEB_PAGE_1 VARCHAR,
+	CONTEXTS_COM_IAB_SNOWPLOW_SPIDERS_AND_ROBOTS_1 VARCHAR,
+	CONTEXTS_NL_BASJES_YAUAA_CONTEXT_1 VARCHAR,
+	constraint EVENT_ID_PK primary key (EVENT_ID)
+);
+
+```
+
+
+
+***
+
+#### **Step 3:** Load the data
+
+3.1 Log into your web interface and click on `Databases` tab.
+
+3.2 Locate the **SAMPLE_EVENTS_BASE** table that you just created and select it.
+
+3.3 Click the `Load Data` button to open the Load Data wizard.
+
+3.4 Select the relevant warehouse from the dropdown list. Click `Next`.
+
+3.5 Within the `Source Files` section select `Load files from your computer` option, and click the `Select Files` button. If you have not saved the sample file provided as an attachment above please do so.
+Navigate to the **SAMPLE_EVENTS.csv** and click the `Upload` then the `Next` button.
+
+3.6 Create a new File Format with the plus (+) symbol beside the dropdown list, give it a name and change the following settings of the default csv file formats:
+- `Header lines to skip`= 1
+- `Field optionally enclosed by`= Double Quote
+
+3.7 Click the `Load` button (no need to alter the Load Options). Loading should take place within a couple of minutes.
+
+For more details please check out the official [Snowflake documentation](https://docs.snowflake.com/en/user-guide/data-load-web-ui.html).
+
+***
+
+#### **Step 4:** Create the **ATOMIC.SAMPLE_EVENTS** table
+
+The Snowplow pipeline creates context fields as arrays however for the web data model to work, these need to be converted to varchars. Run the below DDL statement in your SQL editor:
```

Comment:
Small change here to add a comma, from "...as arrays however for..." to
"as arrays, however for"